actions/git_delete_branch.yaml

```diff
@@ -0,0 +1,34 @@
+---
+name: git_delete_branch
```

Comment:
Just curious, I don't see this action being used anywhere yet - where do we plan to use it?